action/protocol/poll/consortium.go

```diff
@@ -113,7 +114,7 @@ func (cc *consortiumCommittee) CreateGenesisStates(ctx context.Context, sm proto
 		return err
 	}
 	actionCtx.Nonce = consortiumCommitteeContractNonce
-	actionCtx.ActionHash = execution.Hash()
+	actionCtx.ActionHash = consortiumCommitteeContractHash
```

Comment:
this is also improper programming:
execution.Hash() will be all zero, since it is a raw tx not signed

reason this is not an issue: this code is not run on mainnet/testnet now

if you start the chain from 0, this will result in an all-0 hash, colliding with the all-0 hash in staking_committee.go